assets/js/components/coinify/email.component.js

```diff
@@ -0,0 +1,32 @@
+angular
+  .module('walletApp')
+  .component('coinifyEmail', {
+    bindings: {
+      email: '<',
```

Comment:
We don't need to, but I think it's a nicer way of dealing with data.

If we were to do `this.email = Wallet.user.email` when the component is initialized, now `email` is duplicate state. When we update `user.email` we have to remember to update `this.email` as well. If `email` is a binding, that is done automatically for us, and we have more control (if we ever want it) through `$onChanges`.

Could also do `this.user = Wallet.user` when the component is initialized, then `ng-model="$ctrl.user.email"` in the template. But, angular two-way binds by default, so I think this would end up changing `user.email` before we want it to. Also, exposing all of `Wallet.user` to the component seems unnecessary.

What do you think? Is there another way that would be cleaner / easier?